webapp-client/src/stylesheets/ContentPane.css

```diff
@@ -0,0 +1,3 @@
+.wrap {
+	flex-wrap: wrap;
+}
```

Comment:
Where in component code is this class used?

Reply:
in ContentPane.js, it's used to make the tiles wrap around to the next line when there are more than 3